lib/config/options/index.ts

```diff
@@ -1977,6 +1977,13 @@ const options: RenovateOptions[] = [
     type: 'string',
     default: `This PR has been generated by [Renovate Bot](https://github.com/renovatebot/renovate).`,
   },
+  {
+    name: 'repoProblemsHeader',
```

Comment:
shouldn't we use a more general templates string dictionary, so it's later more extendable?